qmule/src/org/qcmg/qmule/FixCutAdapter.java

```diff
@@ -0,0 +1,101 @@
+package org.qcmg.qmule;
+
+import picard.cmdline.CommandLineProgram;
+import org.broadinstitute.barclay.argparser.Argument;
+
+import picard.cmdline.StandardOptionDefinitions;
+import htsjdk.samtools.util.IOUtil;
+import htsjdk.samtools.util.StringUtil;
+import htsjdk.samtools.SAMException;
+import htsjdk.samtools.fastq.FastqConstants;
+
+import htsjdk.samtools.fastq.FastqRecord;
+import htsjdk.samtools.fastq.FastqWriter;
+import htsjdk.samtools.fastq.FastqWriterFactory;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.Date;
+
+public class FixCutAdapter extends CommandLineProgram {
+    static final String USAGE = "A Fastq file may contains empty seqence after cutAdapter,  Here A base N is set to empty sequence with 0 base quality value";
```

Comment:
For clarity in the help; I suggest:
```
USAGE = "When cutadapt finds adapter sequence at the start of a read the default --action=trim results in empty sequence in the output. This tool replaces any empty sequence with a single base N with 0 base quality value.";
```

Reply:
Our "cutadapt" version 1.9" doesn't support the "action" option, but provide option "--trim-n" or "--mask-adapter".  I will put this into the usage. Thanks.